frontend/src/routes/Infrastructure/Clusters/ManagedClusters/CreateCluster/CreateCluster.tsx

```diff
@@ -325,6 +325,36 @@ export default function CreateCluster(props: { infrastructureType: ClusterInfras
       case 'connection':
         setConnectionControl(control)
         break
+      case 'namespace':
+        control.validation.contextTester = (
+          active: string | undefined,
+          _controlData: object,
+          templateObjectMap: { [x: string]: { HostedCluster: { $raw: { metadata: { namespace: any; name: any } } }[] } }
+        ) => {
+          if (infrastructureType === Provider.kubevirt && templateObjectMap['<<main>>'].HostedCluster[0]) {
+            if (managedClusters.length) {
```

Comment:
This check is also not quite in the right place. Even if there are zero managed clusters, we should still not allow the HostedCluster name and namespace to match, because we will be creating a ManagedCluster of the same name.